WordPress/src/main/res/drawable/ic_layout_selection_overlay.xml

```diff
@@ -0,0 +1,11 @@
+<vector android:autoMirrored="true" android:height="230dp"
+    android:viewportHeight="230" android:viewportWidth="148"
+    android:width="148dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#00000000"
+        android:pathData="M4,1L144,1A3,3 0,0 1,147 4L147,226A3,3 0,0 1,144 229L4,229A3,3 0,0 1,1 226L1,4A3,3 0,0 1,4 1z"
+        android:strokeColor="#C9356E" android:strokeWidth="2"/>
```

Comment:
We usually include the width of the icon in its name eg "ic_trash_white_24dp.xml". We might want to consider following this convention for consistency.

We also try to use only colors from the colors.xml - ideally re-use those which are already defined there under the "Semantic" label. Eg for this color we'd use `@color/accent_50`. 

Wdyt?

Reply:
Fixed with 605aa7aa4c65902d1a250d63af40042ff8e8a27b